### ISSUE TITLE
Why use manually-written Conv2dSame over torch.nn.Conv2d(padding='same')?

### ISSUE THREAD

**Issue by HuiningYuan** [NM512/dreamerv3-torch#32]
Hi, I really appreciate this work, just a small questions though.

In this project, the convolutional layers are from a Conv2dSame class that implements 'same' padding manually. 
However, torch.nn.Conv2d does have 'same' padding mode built-in.
Why use the manually-written Conv2dSame over torch.nn.Conv2d(padding='same')?

**Comment by NM512** [comment]
Hello,

The issue is that the stride of torch.nn.Conv2d(padding='same') is limited to values greater than 1. However, according to the paper, we need to set the stride to 2. You can find a similar discussion on this topic [here](https://github.com/pytorch/pytorch/issues/67551).

I hope this information is helpful to you!
